src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/AvroHelper.java

```diff
@@ -168,6 +173,33 @@ private static void convertColumnTypeToAvroType(
     }
   }
 
+  public static Timestamp getUnadjustedTimestamp(ResultSet row, String columnName)
```

Comment:
Please, add javadoc for these 2 methods
Also, this definitely requires a test. Could you add it?

Reply:
Added javadocs.

Also added unit tests. Ran manual tests too:
```
java -cp ExtractionTool_deploy.jar:terajdbc4.jar com/google/cloud/bigquery/dwhassessment/extractiontool/ExtractionTool td-extract --db-address jdbc:teradata://localhost/DBS_PORT=1025,DATABASE=dbc --output tmp_testTimestamp --db-user dbc --db-password dbc --rows-per-chunk 100 --sql-scripts querylogs --qrylog-timerange-start 2022-03-24T12:39:42.500000 --qrylog-timerange-end 2022-03-26T21:52:01.680000 --no-need-querytext --no-need-jdbc-schemas
```
verifying the timestamps are correct.